pandora-client-web/src/components/chatroom/contextMenus/deviceContextMenu.tsx

```diff
@@ -53,7 +53,7 @@ function MoveDeviceMenu({ device, setChatRoomMode, close }: {
 			itemId: device.id,
 		},
 		target: { type: 'roomInventory' },
-		deployment: CloneDeepMutable(device.deployment),
+		deployment: { deployed: true, position: { x: 0, y: 0, yOffset: 0 } },
```

Comment:
Nitpick: Here I would recommend using the existing position to check - the idea is that existing position must already be valid, so we are unlikely to hit some banned value. (if possible _values_ change in the future someone could easily forget the constants in here)